File: 백준/Bronze/1978. 소수 찾기/소수 찾기.c
#include <stdio.h>
#include <stdlib.h>

int main(void) {
	int n, num = 0;
	scanf("%d", &n);
	int* arr = (int*)malloc(sizeof(int) * n);
	for (int i = 0; i < n; i++) {
		scanf("%d", &arr[i]);
		int m = arr[i];
		int hap = 0;
		while (m > 0) {
			if (arr[i] % m == 0) hap++;
			m--;
		}
		if (hap == 2) num++;
	}
	printf("%d", num);
	
							 
	return 0;				 
}	

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
n = int(input())
arr = list(map(int, input().split()[:n]))
total = 0
for i in arr:
    hap = 0
    if (i > 1):   
        for j in range(2, i):
            if(i % j == 0):
                hap += 1
        if(not hap):
            total += 1
print(total)